test/server/clock.test.js

```diff
@@ -0,0 +1,126 @@
+
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+
+const { Clock } = require('../../server/clock')
+
+chai.use(sinonChai)
+const { expect } = chai
+
+const SECOND = 1000
+const MILISECOND = 1
+
+describe('Clock', () => {
+  let systemTimer
+  let clock
+
+  beforeEach(() => {
+    systemTimer = sinon.useFakeTimers()
+    clock = new Clock()
+  })
+
+  afterEach(() => {
+    systemTimer.restore()
+  })
+
+  it('changes his current time to the given time', () => {
+    clock.time = 30
+
+    expect(clock).property('time').to.be.equal(30)
+  })
+
+  it('changes the time to the given time in the start of countdown', () => {
+    clock.startCountdown(50)
+
+    expect(clock).property('time').to.be.equal(50)
+  })
+
+  it('changes the time according to time', () => {
+    clock.startCountdown(50)
+
+    systemTimer.tick(30 * SECOND)
+
+    expect(clock).property('time').to.be.equal(20)
+  })
+
+  it('changes the time stop changing the time after stopping the countdown', () => {
```

Comment:
Dos you mean: " stops changing the time"?

Reply:
Fix it